critiques/pr-14/validation.md

```
@@ -0,0 +1,331 @@
+
+| | `origin/rewrite` | this branch |
+|---|---|---|
+| passed | 23 | **24** |
+| skipped | 801 | **800** |
+| collected | 824 | **824** |
+| collection errors | 0 | **0** |
+
+`passed + skipped == collected` in both. The set difference of passing test ids
+is exactly one test, in one direction:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)